pkg/cluster/hash_test.go

```diff
@@ -36,34 +36,34 @@ func TestHashScaleSetStability(t *testing.T) {
 			{
 				role: api.AgentPoolProfileRoleMaster,
 				// this value should not change
```

Comment:
this ^
I don't think we should "fix" the old versions

Reply:
I think this will be aligned with other PR's, no?

Reply:
i am ok with it..